BDArmory/Weapons/Missiles/MissileLauncher.cs

```diff
@@ -2205,9 +2205,14 @@ void AAMGuidance()
                         loftTerminal = true;
                     }*/
 
-                    if (BDArmorySettings.DEBUG_MISSILES) Debug.Log("[BDArmory.MissileLauncher]: AAM Loft TTGO: [" + TimeToImpact + "]. Currently State: " + loftState + ".");
+                    if (BDArmorySettings.DEBUG_MISSILES) Debug.Log($"[BDArmory.MissileLauncher]: AAM Loft TTGO: [{TimeToImpact:G3}]. Currently State: {loftState}. Fly to: [{aamTarget}]. Target Position: [{TargetPosition}]. Max AoA: [{maxAoA:G3}]");
                 }
-                else // AAM Lead
+                else if (GuidanceMode == GuidanceModes.AAMPure)
+                {
+                    TimeToImpact = Vector3.Distance(TargetPosition, transform.position) / Mathf.Max((float) vessel.srfSpeed, optimumAirspeed);
+                    aamTarget = TargetPosition;
+                }
```

Comment:
I'm not sure what AAMPure does, so no idea. @SuicidalInsanity or @josuenos would have to comment on it.

Reply:
AAMPure is supposed to be pure pursuit, however it was never implemented within MissileLauncher.cs despite being defined as a type of guidance available and having the file parsing section written. I've just added in the functionality so that it actually works, instead of just flying straight forwards. I meant to put this as a separate pull but I accidentally left it in while making the edits.

Reply:
Ah ok, I think it's probably fine to keep it then.